src/Menu/Menu.style.ts

```diff
@@ -9,15 +9,17 @@ const menuCssVariables = css`
 
 export const FloatingMenuContainer = styled.div`
   height: 100%;
-  box-shadow: ${theme.shadow()};
-  border-radius: 4px;
+  padding: 8px 0;
```

Comment:
Why did you change the style ?

Reply:
`box-shadow` and `border-radius` didn't belong here.
As for `padding` goes, since there was a slight space between `Options` and `Select` components that was preventing `Options` to stay opened when hovering.
Thus explaining why I needed to add a little extra space.